ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,175 @@
+
+
+
+## Actions
+
+![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/AddAction.png "[Action]")
+
+Actions are the individual activities being executed in the workflow and are represented by a rectangle.
+
+All actions contain a certain, predefined number of input and output ports, which can be mandatory or optional. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-ports).
+
+ When adding a new action, all compatible input and output ports are included. Mandatory input ports are already displayed and linked to a place, whereas optional ones are displayed with a ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button.
+
+ [comment]: <> (Processes - All places linked to inputs and outputs ports must be provided with one or more pieces of data, also known as tokens, for the actions to be executed and the workflow to be completed. Otherwise, the workflow stops at the point where tokens are missing. Once an action is executed, one or more output data, or tokens, are generated, which in turn become/s input data, or tokens, for the next action. For detailed information about tokens, see...)
+
+When clicking an action, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+
+The action settings side bar is located to the right of the workspace and allows to define the action settings. The following fields are displayed:
+
+  - (Action label and name)  
+  The action label is an action identification tag and can be defined by user. The action name is a system wide predefined denomination of the activity to be performed and is read-only.
+
+  - *Key*  
+   Action key. This field is read-only. The key is required for API access and must be system wide unique.
+
+  - *Label*  
+  Action label. Click this field to edit the action label. The action label is changed both in the settings side bar and in the diagram.
+
+  - *Queue type*  
+  Click the drop-down list to select the applicable queue type. For detailed information about queue types, see [Configure the queue types](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md).
+
+  - *Task event*  
+  Click the drop-down list to select the applicable task event.
+
+  > [Info] This option is linked to the *Tasks* module. It is only displayed if the *Tasks* module is installed and the user has the necessary rights. The *ActindoWorkflow Process Execution Failed* task is preconfigured by default. A list of all existing task events is displayed in the *Events* menu entry of the *Tasks* module.
+
+  - *Configuration*  
+  Depending on the action, there might be configuration options, for instance, defining an email where a receipt or document must be sent. Editing the action configuration requires advanced programming knowledge.
```

Comment:
Add info from gitlab: 
An action may have configuration options, but most of them didn't have